### ISSUE TITLE
Concentrator error when loading/refreshing site

### ISSUE THREAD

**Issue by Nordiii** [death-save/combat-utility-belt#543]
**Describe the bug**
Concentrator throws error when loading the site

**To Reproduce**
1. Activate Concentration with 'Automatically set Concentration' enabled
2. Import any Charakter 
3. use any weapon (mine was cleric dragon born -> mace)
4. reload site 
5. See error

**Expected behavior**
no error

**Screenshots**
```
TypeError: this.objects is null
    createObject https://../scripts/foundry.js:19413
    get object https://.../scripts/foundry.js:8355
    get https://.../scripts/foundry.js:19470
    _onRenderChatMessage https://.../modules/combat-utility-belt/modules/concentrator.js:48
    lightUp https://.../modules/combat-utility-belt/modules/signal.js:192
    _call https://.../scripts/foundry.js:194
    call https://.../scripts/foundry.js:179
    getHTML https://.../scripts/foundry.js:12904
foundry.js:197:15
```

**Environment Info (please complete the following information):**
 - Foundry Version: 0.8.6
 - Game System: dnd5e 1.3.3
 - CUB Version: 1.5.0
 - Browser firefox
 - OS: Win10
 - Server OS: PiBuster (Debian)

**Additional context**
This was tested with only this module enabled.

**Comment by eclarke12** [comment]
Please try again with v1.6 and report back

**Comment by Nordiii** [comment]
It seams that the message just changed a little bit, (test world and my actual game world):

```
TypeError: canvas.tokens is undefined
    _onRenderChatMessage https://lmop.nordiii.de/modules/combat-utility-belt/modules/concentrator.js:48
    lightUp https://.../modules/combat-utility-belt/modules/signal.js:192
    _call https://.../scripts/foundry.js:194
    call https://.../scripts/foundry.js:179
    getHTML https://.../scripts/foundry.js:12950
    _renderBatch https://.../scripts/foundry.js:32162
    _renderInner https://.../scripts/foundry.js:32132
    _render https://.../scripts/foundry.js:2048
    _render https://.../scripts/foundry.js:23186
    _render https://.../scripts/foundry.js:32123
    _render https://.../scripts/foundry.js:22995
    render https://.../scripts/foundry.js:2004
    initializeUI https://.../scripts/foundry.js:4391
    setupGame https://.../scripts/foundry.js:4293
    _initializeGameView https://.../scripts/foundry.js:5117
    _initializeView https://.../scripts/foundry.js:5093
    initialize https://.../scripts/foundry.js:4202
    async* https://.../scripts/foundry.js:53073
    async* https://.../scripts/foundry.js:53055
```

**Comment by eclarke12** [comment]
this is saying `canvas.tokens` is undefined which means it can't find any tokens on the scene. Can you confirm the default scene is empty?

**Comment by Nordiii** [comment]
My scene is NOT empty, neither the test scene nor the actuall campange scene.

I noticed though that when I load the page without console open it does not throw the error.

When the console is open it looks like canvas.tokens is checked when the scene is still loading.

**Comment by eclarke12** [comment]
resolved in #584
